### ISSUE TITLE
Provide a way to deploy new dags from github

### ISSUE THREAD

**Issue by Sedward** [gsemet/kube-airflow#6]
It had previously been discussed on slack that we could use the the git-sync sidecar for updating dags from github. The environment variables have already been added. We could add the git-sync sidecar to the scheduler, worker and web deployments. If this is done, do we make it an option in the values.yaml that has to be enabled?

**Comment by gsemet** [comment]
it is already done, what is the difference with the current git-sync script and settings in values.yaml?

**Comment by Sedward** [comment]
How does it currently work? I haven't used git-sync yet but I was expecting to see a sidecar in the deployments that use it like the example in their repo. 

https://github.com/kubernetes/git-sync/blob/master/demo/config/pod.yaml#L9

**Comment by gsemet** [comment]
Read the read me. It is a gitsync launched in the container. Using a sidecar means sharing a folder. Which is also a valid use case

**Comment by Sedward** [comment]
OK I see it now. Is there a benefit to using this script inside the container vs the sidecar container from the kubernetes org? I thought it was somewhat of an anti-pattern to run more than one process in a container.

**Comment by gsemet** [comment]
On we need to ensure a share path. If you adapt the conf with an optional sidecar gitsync it can be good to have.

**Comment by Jonathan34** [comment]
I see the value `git_sync_enabled` but i can't see where it is used in the charts. which makes me think this chart only support embedded dags via docker image?

**Comment by gsemet** [comment]
yes, it does not work anymore, I did not took the time to restore it, and it has many issues that need to be fixed
